sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -564,3 +969,108 @@ def get_sparsity_stats(embeddings: torch.Tensor) -> dict[str, float]:
             "row_sparsity_mean": torch.mean(sparsity_per_row).item(),
         }
         return results
+
+    @property
+    def max_seq_length(self) -> int:
+        """
+        Returns the maximal input sequence length for the model. Longer inputs will be truncated.
+
+        Returns:
+            int: The maximal input sequence length.
+
+        Example:
+            ::
+
+                from sentence_transformers import SparseEncoder
+
+                model = SparseEncoder("naver/splade-cocondenser-ensembledistil")
+                print(model.max_seq_length)
+                # => 512
+        """
+        return super().max_seq_length
+
+    @max_seq_length.setter
+    def max_seq_length(self, value) -> None:
+        """
+        Property to set the maximal input sequence length for the model. Longer inputs will be truncated.
+        """
+        self._first_module().max_seq_length = value
+
+    def intersection(
+        self,
+        embeddings_1: torch.Tensor,
+        embeddings_2: torch.Tensor,
+    ):
+        """
+        Compute the intersection of two sparse embeddings.
+
+        Args:
+            embeddings_1 (torch.Tensor): First embedding tensor (batch_1, vocab).
```

Comment:
```suggestion
            embeddings_1 (torch.Tensor): First embedding tensor (vocab).
```
The first embedding must be 1-dimensional